test/Typewriter.Test/TestDataJava/com/microsoft/graph/requests/extensions/IOnenotePageForwardRequestBuilder.java

```diff
@@ -28,4 +28,5 @@
      * @return the IOnenotePageForwardRequest instance
      */
     IOnenotePageForwardRequest buildRequest(final java.util.List<? extends com.microsoft.graph.options.Option> requestOptions);
+
```

Comment:
T4 and whitespace management, I feel the pain :)

Reply:
yep parsing the code is hard enough, keeping tabs on spaces (pun intended) is almost impossible.